### PR TITLE
Cloud icon profile changes

### PR DESCRIPTION
**Pull request by huwshimi**
Add cloud icons to the model switcher and profile model list. Also updates the columns of the profile model list.

Fixes: canonical-web-and-design/juju-squad/issues/597.
Fixes: canonical-web-and-design/juju-squad/issues/510.
Fixes: canonical-web-and-design/juju-squad/issues/607.
Fixes: canonical-web-and-design/juju-squad/issues/579.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- visit /new (otherwise you'll be redirected to jaas.ai
- load your profile
- check that the model list has no last accessed time and has cloud icons like this: https://app.zeplin.io/project/5c5871a14e1897355935afb1/screen/5c6bc4e67fee003c05c7928b
- close your profile
- in the model switcher check that the list has no last accessed time and has cloud icons like this: https://app.zeplin.io/project/5c5871a14e1897355935afb1/screen/5c6bc4e9ae27d83c119b4ffe

**Comment by clagom** [comment]
Hey @huwshimi thank you it looks good. I just wonder, in the profile page it's missing units/pods column now?

**Comment by clagom** [comment]
Sorry @huwshimi ignore my previous comment, units and pods are inside of the row when you open it:


design:
https://zpl.io/2EBw9jM

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,6 @@ PYTEST := bin/py.test
 GUISRC := jujugui/static/gui/src
 GUIBUILD := jujugui/static/gui/build
 ASSETS_DIR := $(GUISRC)/app/assets
-SVG_SPRITE_SOURCE_DIR := $(GUISRC)/app/assets/svgs
```
Comment (hatched) [inline Makefile]: The `Makefile` isn't being used any longer. I actually left it here just for this one target :) I'll look into whether we can remove it wholesale now.

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "lint-js": "eslint --rulesdir eslint-rules gui/src",
     "lint": "yarn run lint-js && yarn run lint-python",
     "serve": "./entrypoint 0.0.0.0:$PORT",
+    "svg-sprite": "yarn add svg-sprite@1.3.6 && svg-sprite --dest ./static/gui/build/app/assets --stack ./static/assets/svgs/*.svg && yarn remove svg-sprite",
```
Comment (hatched) [inline package.json]: nice

**Pull request merged**